cms/static/cms/js/modules/cms.modal.js

```diff
@@ -584,7 +582,11 @@ $(document).ready(function () {
 				// when the window has been changed pressing the blue or red button, we need to run a reload check
 				// also check that no delete-confirmation is required
 				if(that.saved && !contents.find('.delete-confirmation').length) {
-					that.reloadBrowser(window.location.href, false, true);
+					if(that.options.onClose){
```

Comment:
```if (that.options.onClose) {``` (though i see that this format is consistent with what's currently in the file so maybe it's better to change en masse in a later branch).